test/integration/targets/win_module_utils/library/camel_conversion_test.ps1

```diff
@@ -40,7 +40,7 @@ foreach ($entry in $output_dict.GetEnumerator()) {
     $key = $entry.Name
     $value = $entry.Value
 
-    $type = $value.GetType()
+    $value.GetType()
```

Comment:
If we aren't using `$type` just remove the call, we shouldn't be outputting anything to the output pipeline unless it's the return json.